site2/docs/administration-pulsar-shell.md

```diff
@@ -0,0 +1,174 @@
+---
+id: administration-pulsar-shell
+title: Pulsar Shell
+sidebar_label: "Pulsar Shell"
+---
+
+````mdx-code-block
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+````
+
+
+Pulsar shell is a fast and flexible shell for Pulsar cluster management, messaging, and more.
+It's great for quickly switching between different clusters, and can modify cluster or tenant configurations in an instant.
+
+
+# Features
+- Administration: find all the [Admin API](admin-api-overview.md) features under  the `admin` command .
+- Client: find all the [pulsar-client](reference-cli-tools#pulsar-client.md) features under the `client` command.
+
+
+# Installation
+
+Download the tarball from the [download page](https://pulsar.apache.org/download) and extract it.
+
+```shell
+wget https://archive.apache.org/dist/pulsar/pulsar-@pulsar:version@/apache-pulsar-shell-@pulsar:version@-bin.tar.gz
+tar xzvf apache-pulsar-shell-@pulsar:version@-bin.tar.gz
+cd apache-pulsar-shell-@pulsar:version@-bin.tar.gz
+```
+
+Now you can enter Pulsar shell's interactive mode:
+
+```shell
+$ ./bin/pulsar-shell
+Welcome to Pulsar shell!
+  Service URL: pulsar://localhost:6650/
+  Admin URL: http://localhost:8080/
+
+Type help to get started or try the autocompletion (TAB button).
+Type exit or quit to end the shell session.
+
+default(localhost)> 
+```
+
+
+# Connect to your cluster
+
+By default, the shell will try to connect to a local Pulsar instance.
```

Comment:
```suggestion
By default, the shell tries to connect to a local Pulsar instance.
```

We use the simple present tense according to the [Pulsar Writing Style Guide](https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#heading=h.j0ohedastxh).